### ISSUE TITLE
Configuring a CLA with a different user does not share Contributor list

### ISSUE THREAD

**Issue by schloerke** [cla-assistant/cla-assistant#804]
## URL to the linked Repository or Organization

Correct: https://github.com/rstudio/bslib
Missing contributors: https://github.com/rstudio/fontawesome

## Steps to reproduce the problem

1. @trestletech configures a repo, we trigger the CLA via a Pull request. Works as expected.
2. @schloerke configures a new repo following the same steps as @trestletech, I trigger the CLA via https://github.com/rstudio/fontawesome/pull/78, but the repo does not know about the prior CLA submissions.

## What is the expected result?

CLAs added to repos that point to the same shared gist should inherit the existing contributors independent of who adds the configuration.

## What happens instead?

The repo added by @schloerke is being treated as a fresh CLA and does not inherit the existing contributor list.

## Any other information? (attach screenshot if possible)

![Screen Shot 2021-11-22 at 10 38 21 AM](https://user-images.githubusercontent.com/93231/142890335-910315f8-015b-471a-bf80-3084bbb451fd.png)

## Used browser and browser version?

macOS Chrome, 95.0.4638.69

**Comment by Shegox** [comment]
Hi, sorry for the long wait. 

I had today time and could confirm that behavior in my first test. 

After checking the data, I found that this is actually caused by something different. 
When you select the CLA from the dropdown the URL used is something like `https://gist.github.com/8e7f48c73e2635b253bd5914bbfeab05` while when copying it from the browser it is expanded and includes the username like `https://gist.github.com/Shegox/8e7f48c73e2635b253bd5914bbfeab05`. 

Looking at the database the same happened here, all 32 repos except the "fontawesome" repo are using the gist url without name. To solve this in the short-term, the "fontawesome" repo should use the url without the name in it.

I created #806 to track a proper fix on our side for this. 

Thank you very much for opening this issue and the detailed description. The description and screenshots helped a lot in reproducing and identifying the root issue.
